msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AbstractClientApplicationBase.java

```diff
@@ -643,6 +648,18 @@ public T azureRegion(String val) {
             return self();
         }
 
+        /** Historically, MSAL would connect to a central endpoint located at
```

Comment:
Rather than have a long comment here, it might be better to have a wiki page describing this behavior and make this comment briefer with an aka.ms link to that page.

Reply:
sure, let's talk more about this.

Reply:
FWIW, I use one-or-two-paragraphs long comments all the time, because in-place comment gives better locality to next maintainer (or a future you). And it does not rely on an external link thus insusceptible to future bad-link problem (which can happen to aka.ms links, too). Of course, the long comment itself can end with a link to a long long specs or Microsoft Docs link, it is OK for those links to go stale, because the gist is already captured inside the comment.

Besides, will this long comment be automatically used to generate the API docs? If so, then it  is even preferable to write long description here.

Reply:
Yes for the information presented here, I would prefer having these lines here instead of an aka.ms link. If the information is a page long, then it makes sense to move this to another link.